src/components/reports/ReportPieChart.vue

```diff
@@ -169,7 +169,7 @@ export default {
       const width = 80 * 6 * 3;
       const height = 260 * Math.ceil(data.length / 3);
       const svg = d3
-        .select(`#${props.id}`)
+        ?.select(`#${props.id}`)
         .append('svg')
```

Comment:
```suggestion
        .select(`#${props.id}`)
        ?.append('svg')
```
I am pretty sure `d3` would be defined in this case. So using optional chaining after select would be better. Can you reply with sentry issue link?